gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -336,15 +339,15 @@ If `flex-gateway.service` has a status of `active`, Flex Gateway is successfully
 
 // end::k8s-install-flex-helm-chart-title[]
 // tag::k8s-install-flex-helm-chart-intro-connected[]
-Use Helm to deploy Flex Gateway to a node in your cluster and connect to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
+
+Register Flex Gateway, and use Helm to deploy Flex Gateway to a node in your cluster and connect to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
```

Comment:
```suggestion
Register Flex Gateway, then use Helm to deploy Flex Gateway to a node in your cluster and connect to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
```
I wonder if it should be "then"? Having two instances of "and" in the same sentence was tripping me up a little.

Reply:
I think it would be two independent clauses in that case. Maybe "and then"?